docs/api.md

```diff
@@ -0,0 +1,908 @@
+<td align="left"><div class="first-page">
+<p>Promise from an API request</p>
+</div></td>
+</tr>
+</tbody>
+</table>
+
+#### Example
+
+    model('posts').first({name: 'some post'}).then(function(post) {
+      console.log(post._id);
+    });
+    => 42
+
+
+`$collection`
+=============
+
+<span class="hint">object in module `ur` </span>
+
+Description
+-----------
+
+Methods available on model collections
+
+You can use collection methods to deal with a bunch of instances together. This allows you to have powerful and expressive methods on collections.
+
+You can specify custom collection methods:
+
+     yourApp.config(function(modelProvider) {
+       modelProvider.model('posts', {
+         $collection: {
+           $hasArchived: function() {
+             return !angular.isUndefined(_.find(this, { archived: true }));
+           }
+         },
+       });
+     });
+
+Methods
+-------
+
+### add(object, data)
+
+Saves the `object` with `data`
+
+##### Parameters
+
+<table>
+<colgroup>
+<col width="33%" />
+<col width="33%" />
+<col width="33%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th align="left">Param</th>
+<th align="left">Type</th>
+<th align="left">Details</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td align="left">object</td>
+<td align="left"><a href="">object</a></td>
+<td align="left"><div class="add-page">
+<p>Object to persist data onto</p>
+</div></td>
+</tr>
+<tr class="even">
+<td align="left">data
+<div>
+<em>(optional)</em>
+</div></td>
+<td align="left"><a href="">object</a></td>
+<td align="left"><div class="add-page">
+<p>Data to persist onto the object</p>
+</div></td>
+</tr>
+</tbody>
+</table>
+
+##### Returns
+
+<table>
+<colgroup>
+<col width="50%" />
+<col width="50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td align="left"><a href="">boolean</a></td>
+<td align="left"><div class="add-page">
+<p>true if a $link to <code>name</code> exists on this instance</p>
+</div></td>
+</tr>
+</tbody>
+</table>
+
+### remove(index)
+
+Find `index` and delete it from the API, then remove it from the collection
+
+##### Parameters
+
+<table>
+<colgroup>
+<col width="33%" />
+<col width="33%" />
+<col width="33%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th align="left">Param</th>
+<th align="left">Type</th>
+<th align="left">Details</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td align="left">index</td>
+<td align="left"><a href="">number</a><a href="">object</a></td>
+<td align="left"><div class="remove-page">
+<p>Either the index of the item in the collection to remove, or the object itself, which will be searched for in the collection</p>
+</div></td>
+</tr>
+</tbody>
+</table>
+
+##### Returns
+
+<table>
+<colgroup>
+<col width="50%" />
+<col width="50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td align="left"><a href="">object</a></td>
+<td align="left"><div class="remove-page">
+<p>Promise from the API</p>
+</div></td>
+</tr>
+</tbody>
+</table>
+
+
+`$get`
```

Comment:
I'm not sure it makes sense to document this, since it's an artifact of how Angular handles the interaction between/definitions of the provider and the factory itself.